ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessStatus.scala

```diff
@@ -54,12 +49,17 @@ object ProcessStatus {
       errorMessage
     )
 
-  def create(processState: ProcessState, expectedDeploymentVersion: Option[Long]): ProcessStatus = {
-    val versionMatchMessage = (processState.version, expectedDeploymentVersion) match {
-      case (Some(stateVersion), Some(expectedVersion)) if stateVersion.versionId == expectedVersion => None
-      case (Some(stateVersion), Some(expectedVersion)) if processState.isDeployed => Some(s"Process deployed in version ${stateVersion.versionId} (by ${stateVersion.user}), expected version $expectedVersion")
+  def isStateAfterDeploy(processState: ProcessState): Boolean =
```

Comment:
What does it mean state is after deploy? Maybe some better verb will help like `isFollowingDeployAction`. Please move it to `ProcessState`. It is one more method describing lifecycle that we should remember about if we want to make this lifecycle more elastic.

Reply:
Changed to isFollowingDeployAction.

Reply:
What about 2nd part:

> Please move it to ProcessState. It is one more method describing lifecycle that we should remember about if we want to make this lifecycle more elastic.

Reply:
You'r right :) Done.